### ISSUE TITLE
Accessibility issues on the login screen

### ISSUE THREAD

**Issue by plannero** [umbraco/Umbraco-CMS#14517]
### Which Umbraco version are you using? (Please write the *exact* version, example: 10.1.0)

10.4.0

### Bug summary

**1. Contrast**
Login screen has insufficient text contrast on login button as well as "show password" and "forgotten password?".
Contrast looks a little better when not Login button not disabled, but still not OK according to WCAG 2.1.
<img width="588" alt="contrast_login" src="https://github.com/umbraco/Umbraco-CMS/assets/2157085/bcd58560-83fc-4b7d-a885-40d6a89d1f14">

**2. Umbraco logo missing alt text**
Screen reader should not need to fallback to reading the file name (even if that is somewhat readable). Just add alt="Umbraco logo" or similar.
<img width="536" alt="logo_alt" src="https://github.com/umbraco/Umbraco-CMS/assets/2157085/6faaca2d-f66b-4d2c-a4ca-5960956df140">

**3. Very nice, but not very informative h1** 
The formulation I find in h1 has a nice tone, and I believe you should keep it (because the title element + the layout kind of provide the needed message). But formally it might be considered a breach of accessibility standards for clear and informative headings. https://www.w3.org/TR/WCAG21/#headings-and-labels 
<img width="590" alt="thunder" src="https://github.com/umbraco/Umbraco-CMS/assets/2157085/36ebc7e8-6605-4605-b445-7a465354355d">

 

 

### Steps to reproduce

Go to login screen. Observe.

**Comment by elit0451** [comment]
Thank you for reaching out @plannero and making us aware of these accessibility issues. 💪 

In order to work on those issue, we need each problem reported in a separate GitHub issue. Can you amend this?
We also encourage you to submit a PR with your suggestions 😊

**Comment by nielslyngsoe** [comment]
There has been put an effort into improving accessibility in newer version s of Umbraco. And we are continuing to improve. This perticular issue is not relevant any longer and therefor I will close it. Upgrade to latest version a get amuch better experience.

Thanks in advance, and thanks for the repport, please bring similar topics regarding the latest version.
